src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery extends Ball {
+    Lottery() {
+        super();
+    }
+
+    Lottery(String color, int number) {
+        super(color, number);
+    }

```

Comment:
This is not required after you remove improper inheritance
```suggestion
```

Reply:
ok, done,